### PR TITLE
NOISSUE - Format log messages from agent

### PR DESCRIPTION
**Pull request by SammyOina**
# What type of PR is this?

## What does this do?
- **New Features**
	- Enhanced logging mechanism for agent events and logs, providing structured and informative log entries.
  
- **Bug Fixes**
	- Adjusted Go version to ensure compatibility with the application’s dependencies.

### Notes

```shell
journalctl -u cocos-manager -f -n 150 -o json | jq '                                                                                                                                                                        INT ✘  20s  
{
  __REALTIME_TIMESTAMP,
  _SYSTEMD_UNIT,
  _HOSTNAME,
  PRIORITY,
  MESSAGE: (.MESSAGE | fromjson)
}'
```

image

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,6 +1,6 @@
 module github.com/ultravioletrs/cocos
 
-go 1.23.0
+go 1.22.5
```
Comment (WashingtonKK) [inline go.mod:4]: What's the issue with buildroot that requires us to downgrade go version?

Comment (SammyOina) [reply go.mod:4]: https://github.com/buildroot/buildroot/tree/master/package/go

Comment (drasko) [reply go.mod:4]: Why is this downgraded?

**Review by drasko — CHANGES_REQUESTED** [review]

File: go.mod (hunk 1 above)
Comment (SammyOina) [reply go.mod:4]: buildroot does not support go 1.23 yet

**Review by WashingtonKK — APPROVED** [review]

**Pull request merged**